packages/serverless/README.md

```diff
@@ -53,6 +53,19 @@ Sentry.AWSLambda.init({
 });
 ```
 
+#### Integrate Sentry using internal extension
+
+Another and much simpler way to integrate Sentry to your AWS Lambda function is to add an official layer.
+
+1. Choose Layers -> Add Layer.
+2. Specify an ARN: `arn:aws:lambda:us-west-1:TODO:layer:TODO:VERSION`.
+3. *(optional)* Go to Environment variables and add:
+   - `NODE_OPTIONS`: `-r @sentry/serverless/dist/auto`.
```

Comment:
Using such generic name as `auto` is imo not enough in a package where we keep multiple vendors together. It's impossible to tell whether `serverless/dist/auto` refers to AWS, GCF or another serverless service.